File: testprojects/java/accuracy/technology/direct/subclass/CallInstanceSubSubClassDOA.java
package technology.direct.subclass;

import technology.direct.dao.CallInstanceSuperClassDAO;

public class CallInstanceSubSubClassDOA extends CallInstanceSubClassDOA {
	
	public CallInstanceSubSubClassDOA() {
		};
	
	public CallInstanceSuperClassDAO MethodSuperClassInstance;
	
	public void test(){
		MethodSuperClassInstance.MethodOnSuperClass();
	}
}

File: testresources/csharp/benchmark/Technology/Direct/Subclass/CallInstanceSubSubClassDOA.cs
using Technology.Direct.Dao;

namespace Technology.Direct.Subclass
{
	public class CallInstanceSubSubClassDOA : CallInstanceSubClassDOA
	{
		public CallInstanceSuperClassDAO MethodSuperClassInstance;

		public CallInstanceSubSubClassDOA()
		{
		}

		public override void Test()
		{
			MethodSuperClassInstance.MethodOnSuperClass();
		}
	}
}
